wpcli-localwp-setup

```diff
@@ -83,7 +83,10 @@ echo "${cyan}Creating...${normal} ${php_path}"
 echo
 cat >"${php_path}" <<EOL
 <?php
-define('DB_HOST', 'localhost:${socket}');
+
+if ('/Applications/Local.app/Contents/Resources/extraResources/bin/wp-cli/config.yaml' !== getenv('WP_CLI_CONFIG_PATH')){
```

Comment:
Yeah, that's Mac specific, so it might be better to prompt the user for the environment variable and value to use here.